source/common/grpc/grpc_web_filter.h

```diff
@@ -0,0 +1,54 @@
+#ifndef SOURCE_COMMON_GRPC_GRPC_WEB_FILTER_H_
+#define SOURCE_COMMON_GRPC_GRPC_WEB_FILTER_H_
+
+#include "envoy/http/filter.h"
+
+#include "common/buffer/buffer_impl.h"
+#include "common/grpc/codec.h"
+
+namespace Envoy {
+namespace Grpc {
+
+class GrpcWebFilter : public Http::StreamFilter {
```

Comment:
Add docs/configuration/http_filters/grpc_web_filter.rst with a comment point to it. (By follow the same style as grpc_http1_bridge_filter). Let me know if you want to put more details in the h/cc file.